docker/entrypoint.sh

```diff
@@ -304,6 +300,10 @@ case "$1" in
         # Since these tests use the same cached state, a state problem in the first test can fail the second test.
         run_cargo_test "${ENTRYPOINT_FEATURES}" "--test-threads" "1" "fully_synced_rpc_"
 
+      elif [[ "${TEST_LWD_INTEGRATION}" -eq "1" ]]; then
+        # Test launching lightwalletd with an empty lightwalletd and Zebra state.
+        run_cargo_test "${ENTRYPOINT_FEATURES}" "lightwalletd_integration"
```

Comment:
The `lightwalletd_integration` test is the one that doesn't use a cached state. It looks like `lightwalletd_update_sync` is the intended test.

```suggestion
        run_cargo_test "${ENTRYPOINT_FEATURES}" "lightwalletd_update_sync"
```

Reply:
We already run the update here:
https://github.com/ZcashFoundation/zebra/pull/9038/files#diff-4f5cabe26761257a4d685a6edc7a43e0fe0f78762f50eeb48530f2bd3b3ee7caL317

And we trigger it from CI here:
https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/sub-ci-integration-tests-gcp.yml#L354